### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by robdodson**
Before raising a PR and it getting merged, make sure you have completed the following:

### Things to do:

- [x] Add podcasts in appropriate categories in ascending order of podcast name
- [x] If there is no matching category, add a new category to the list in ascending order of category name
- [x] Add a brief description of the podcast
- [x] Add the frequency of episodes (Check the list for examples)

Thanks for your contributions and being awesome :) Cheers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,3 +294,7 @@ List of podcasts which are helpful for software engineers/programmers.
 - [NodeUp](http://nodeup.com/)
 
 - [Syntax](https://syntax.fm/)
+
+- [HTTP203](https://developers.google.com/web/shows/http203/podcast/)
```
Comment (rShetty) [inline README.md]: Alphabetical order?

Comment (robdodson) [reply README.md]: oops! i'll fix that

**Comment by robdodson** [comment]
should be fixed now

**Comment by rShetty** [comment]
Thank you @robdodson  👍

**Pull request merged**